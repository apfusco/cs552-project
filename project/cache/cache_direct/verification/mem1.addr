0 1 256 0
0 1 258 0
0 1 260 0
0 1 262 0
0 1 264 0
0 1 266 0
