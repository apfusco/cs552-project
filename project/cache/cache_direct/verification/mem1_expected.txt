0 1 256 0 --> MISS
0 1 258 0 --> HIT
0 1 260 0 --> HIT
0 1 262 0 --> HIT
0 1 264 0 --> MISS
0 1 266 0 --> HIT
